package.json

```diff
@@ -1,15 +1,24 @@
 {
-	"license": "MIT",
-	"dependencies": {
-		"@typescript-eslint/eslint-plugin": "^5.51.0",
-		"@typescript-eslint/parser": "^5.51.0",
-		"eslint": "^8.35.0",
-		"eslint-config-prettier": "^8.6.0",
-		"eslint-plugin-prettier": "^4.2.1",
-		"prettier": "^2.8.4",
-		"typescript": "^4.9.5"
-	},
-	"scripts": {
-		"format": "prettier --write .eslintrc.cjs && eslint --quiet --fix --ext .cjs,.ts .eslintrc.cjs"
-	}
-}
+  "author": "Hugging Face",
+  "repository": "github:huggingface/huggingface.js",
+  "dependencies": {
+    "@changesets/changelog-github": "^0.4.8",
+    "@changesets/cli": "^2.26.0",
+    "@typescript-eslint/eslint-plugin": "^5.51.0",
+    "@typescript-eslint/parser": "^5.51.0",
+    "eslint": "^8.35.0",
+    "eslint-config-prettier": "^8.6.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "prettier": "^2.8.4",
+    "typescript": "^4.9.5"
+  },
+  "scripts": {
+    "changeset:release": "pnpm run build && changeset publish",
+    "changeset:version": "changeset version && pnpm install",
+    "build": "pnpm --filter './packages/*' build",
+    "test": "pnpm --filter './packages/*' test",
```

Comment:
I think it's useful to keep these in the root package.json so that running a build for all packages is a simple `pnpm build` command. Same for test.

For example the build script is being used in one of the changesets script right now. 

It's best if there's a build, test, lint and typecheck commands in root and in each package individually but more importantly in root. This way `pnpm build|test|lint|typecheck` can each be invoked from root and if someone's working inside a specific package they can invoke from there too. In CI you'd want to have a workflow that runs on pull requests like you already do but it should run the root lint, typecheck, test, and build commands (in sequence) as opposed to running them for specific packages individually in separate workflows

Reply:
Sure - but those can be a separate PR (which also updates the root's README), it's not related to this one